src/AutorestSwift/Code Generator/UsageSchema.swift

```diff
@@ -28,5 +28,5 @@ import Foundation
 
 /// Protocol that delinates models with a usage property
 protocol UsageSchema {
-    var usage: [SchemaContext] { get }
+    var usage: [SchemaContext]? { get }
```

Comment:
This is an issue I found in Autorest. It sends us an ObjectSchema which does not have `usage` and `serializationFormats`

https://github.com/Azure/autorest/issues/3708

Above is changed to work around the Model4 issue. This issue was hard to debug as AutorestSwift just failed to generate any code. It took some time to find out it is because of missing  `usage` and `serializationFormats`. I have filed an issue to AutoRest